lib/widgets/input_field.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:podcasty/models/input_appearance.dart';
+import 'package:podcasty/values/app_colors.dart';
+import 'package:podcasty/values/enums.dart';
+import 'package:podcasty/values/string_constants.dart';
+
+class InputField extends StatelessWidget {
+  final String hintText;
+  final IconData? suffixIcon;
+  final bool obscureText;
+  final TextInputType keyboardType;
+  final InputFieldAppearance appearance;
+  final Color? iconColor;
```

Comment:
Please avoid making variables nullable if you are passing default value.

Reply:
Resolved